### PR TITLE
Fix/az/check media types

### PR DESCRIPTION
**Pull request by azaxarov**

### CODE REVIEW

File: src/main/java/no/fdk/utils/LanguageUtils.java [hunk 1]
```diff
@@ -5,6 +5,9 @@
 
 public class LanguageUtils {
 
+    private LanguageUtils() {
+    }
```
Comment (Chlenix) [inline src/main/java/no/fdk/utils/LanguageUtils.java]: lombok's `@NoArgsConstructor`?

File: src/main/java/no/fdk/utils/GraphUtils.java [hunk 2]
```diff
@@ -11,6 +11,9 @@
 
 public class GraphUtils {
 
+    private GraphUtils() {
+    }
```
Comment (Chlenix) [inline src/main/java/no/fdk/utils/GraphUtils.java]: lombok's `@NoArgsConstructor`?

**Review by Chlenix — APPROVED** [review]

File: src/main/java/no/fdk/utils/GraphUtils.java (hunk 2 above)
Comment (Chlenix) [reply src/main/java/no/fdk/utils/GraphUtils.java]: Just noticed it was private default. In that case you can:
`@NoArgsConstructor(extraPrivate = true)`

Comment (azaxarov) [reply src/main/java/no/fdk/utils/GraphUtils.java]: Nice one!

**Pull request merged**